pyramid/scripts/pcreate.py

```diff
@@ -99,9 +106,13 @@ def output_path(self):
     def project_vars(self):
         output_dir = self.output_path
         project_name = os.path.basename(os.path.split(output_dir)[1])
-        pkg_name = _bad_chars_re.sub(
-            '', project_name.lower().replace('-', '_'))
-        safe_name = pkg_resources.safe_name(project_name)
+        if self.options.package_name is None:
+            pkg_name = _bad_chars_re.sub(
+                '', project_name.lower().replace('-', '_'))
```

Comment:
Is this code not run on the project name if `--package-name` is not specified? It seems the `pkg_name` should be normalized in one spot.